operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -737,6 +741,10 @@ public boolean keyReplaced() {
         return renewalType == RenewalType.REPLACE_KEY;
     }
 
+    public boolean keyCreated() {
+        return renewalType == RenewalType.CREATE;
```

Comment:
This is mostly stylistic but I believe `.equals` is generally preferred to `==` as we can choose to override how we compare objects and can avoid object hashes not matching https://stackoverflow.com/questions/7520432/what-is-the-difference-between-and-equals-in-java
99% of the time it doesn't matter but thought it was worth a mention.